easybuild/easyconfigs/n/NextClip/NextClip_scripts_shebang.patch

```diff
@@ -0,0 +1,60 @@
+diff --git a/scripts/index_bwa.sh b/scripts/index_bwa.sh
+index 2862e42..472d7f1 100755
+--- a/scripts/index_bwa.sh
++++ b/scripts/index_bwa.sh
+@@ -1,4 +1,4 @@
+-#!/bin/bash
++#!/usr/bin/env bash
+ 
+ source bwa-0.6.1
+ 
+diff --git a/scripts/nextclip_index_reference.pl b/scripts/nextclip_index_reference.pl
+index 03810b3..b7f7649 100755
+--- a/scripts/nextclip_index_reference.pl
++++ b/scripts/nextclip_index_reference.pl
+@@ -1,4 +1,4 @@
+-#!/usr/bin/perl -w
++#!/usr/bin/env perl -w
```

Comment:
Does this still work? I'm not sure if passing options to `perl` in a shebang line is possible...